Compiling failed on rolling

See the error log

```
make: Entering directory '/home/runner/work/rclnodejs/rclnodejs/build'
  CXX(target) Release/obj.target/rclnodejs/src/executor.o
  CXX(target) Release/obj.target/rclnodejs/src/addon.o
  CXX(target) Release/obj.target/rclnodejs/src/handle_manager.o
  CXX(target) Release/obj.target/rclnodejs/src/rcl_action_bindings.o
  CXX(target) Release/obj.target/rclnodejs/src/rcl_bindings.o
  CXX(target) Release/obj.target/rclnodejs/src/rcl_handle.o
  CXX(target) Release/obj.target/rclnodejs/src/rcl_lifecycle_bindings.o
  CXX(target) Release/obj.target/rclnodejs/src/rcl_utilities.o
  CXX(target) Release/obj.target/rclnodejs/src/shadow_node.o
In file included from /opt/ros/rolling/include/rcl/rcl/client.h:30,
                 from /opt/ros/rolling/include/rcl/rcl/wait.h:28,
                 from ../src/executor.hpp:18,
                 from ../src/executor.cpp:15:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/executor.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from /opt/ros/rolling/include/rcl/rcl/rcl.h:77,
                 from ../src/rcl_utilities.cpp:17:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/rcl_utilities.o] Error 1
In file included from /opt/ros/rolling/include/rcl/rcl/rcl.h:77,
                 from ../src/rcl_action_bindings.hpp:19,
                 from ../src/rcl_action_bindings.cpp:15:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
In file included from /opt/ros/rolling/include/rcl/rcl/rcl.h:77,
                 from ../src/rcl_action_bindings.hpp:19,
                 from ../src/addon.cpp:18:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /opt/ros/rolling/include/rcl/rcl/client.h:30,
                 from /opt/ros/rolling/include/rcl/rcl/graph.h:35,
                 from ../src/rcl_bindings.hpp:19,
                 from ../src/rcl_bindings.cpp:15:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /opt/ros/rolling/include/rcl/rcl/client.h:30,
                 from /opt/ros/rolling/include/rcl/rcl/graph.h:35,
                 from ../src/rcl_bindings.hpp:19,
                 from ../src/rcl_lifecycle_bindings.hpp:20,
                 from ../src/rcl_lifecycle_bindings.cpp:15:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
compilation terminated.
In file included from /opt/ros/rolling/include/rcl/rcl/client.h:30,
                 from /opt/ros/rolling/include/rcl/rcl/wait.h:28,
                 from ../src/executor.hpp:18,
                 from ../src/shadow_node.hpp:24,
                 from ../src/shadow_node.cpp:15:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
compilation terminated.
In file included from /opt/ros/rolling/include/rcl/rcl/client.h:30,
                 from /opt/ros/rolling/include/rcl/rcl/wait.h:28,
                 from ../src/handle_manager.hpp:19,
                 from ../src/handle_manager.cpp:15:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/rcl_action_bindings.o] Error 1
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/rcl_lifecycle_bindings.o] Error 1
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/addon.o] Error 1
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/handle_manager.o] Error 1
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/rcl_bindings.o] Error 1
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/shadow_node.o] Error 1
In file included from /opt/ros/rolling/include/rcl/rcl/rcl.h:77,
                 from ../src/rcl_handle.cpp:18:
/opt/ros/rolling/include/rcl/rcl/node.h:36:10: fatal error: type_description_interfaces/srv/get_type_description.h: No such file or directory
   36 | #include "type_description_interfaces/srv/get_type_description.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [rclnodejs.target.mk:166: Release/obj.target/rclnodejs/src/rcl_handle.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/opt/hostedtoolcache/node/16.20.2/x64/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:201:[23](https://github.com/RobotWebTools/rclnodejs/actions/runs/5996467064/job/16261048428#step:6:24))
gyp ERR! stack     at ChildProcess.emit (node:events:513:[28](https://github.com/RobotWebTools/rclnodejs/actions/runs/5996467064/job/16261048428#step:6:29))
gyp ERR! stack     at Process.ChildProcess._handle.onexit (node:internal/child_process:[29](https://github.com/RobotWebTools/rclnodejs/actions/runs/5996467064/job/16261048428#step:6:30)3:12)
gyp ERR! System Linux 5.15.0-1041-azure
gyp ERR! command "/opt/hostedtoolcache/node/16.20.2/x64/bin/node" "/opt/hostedtoolcache/node/16.20.2/x64/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "-j" "16" "rebuild"
gyp ERR! cwd /home/runner/work/rclnodejs/rclnodejs
gyp ERR! node -v v16.20.2
make: Leaving directory '/home/runner/work/rclnodejs/rclnodejs/build'
gyp ERR! node-gyp -v v9.1.0
gyp ERR! not ok 
npm ERR! code 1
npm ERR! path /home/runner/work/rclnodejs/rclnodejs
npm ERR! command failed
npm ERR! command sh -c -- npm run rebuild

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2023-08-28T06_41_00_[31](https://github.com/RobotWebTools/rclnodejs/actions/runs/5996467064/job/16261048428#step:6:32)1Z-debug-0.log
Error: Process completed with exit code 1.
```

--- a/binding.gyp
+++ b/binding.gyp
@@ -90,6 +90,7 @@
                     "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/g, '/include/lifecycle_msgs/ ') + '/include/lifecycle_msgs/')\")",
                     "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/g, '/include/rosidl_runtime_c/ ') + '/include/rosidl_runtime_c/')\")",
                     "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/g, '/include/rosidl_dynamic_typesupport/ ') + '/include/rosidl_dynamic_typesupport/')\")",
+                    "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/:/g, '/include/type_description_interfaces/ ') + '/include/type_description_interfaces/')\")",
                   ],
                 }
               ],
@@ -138,6 +139,7 @@
                     "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/;/g, '/include/rcl_lifecycle ').replace(/\\\/g, '/') + '/include/rcl_lifecycle')\")",
                     "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/;/g, '/include/lifecycle_msgs ').replace(/\\\/g, '/') + '/include/lifecycle_msgs')\")",
                     "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/;/g, '/include/rosidl_dynamic_typesupport ').replace(/\\\/g, '/') + '/include/rosidl_dynamic_typesupport')\")",
+                    "<!@(node -e \"console.log(process.env.AMENT_PREFIX_PATH.replace(/;/g, '/include/type_description_interfaces/ ').replace(/\\\/g, '/') + '/include/type_description_interfaces/')\")",
                   ],
                 }
               ]

